modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -97,11 +97,11 @@ spec:
 +
 [source,terminal]
 ----
-$ oc get -o jsonpath=‘{.status.infrastructureName}{“\n”}’ infrastructure cluster
+$ oc get -o jsonpath="{.status.infrastructureName}{'\n'}" infrastructure cluster
 ----
 <2> Specify the infrastructure ID, role node label, and zone.
 <3> Specify the role node label to add.
```

Comment:
```suggestion
<3> Specifies the role node label to add.
```

Reply:
Using "Specifies" is not required here. Because, it's assigned to a YAML line `machine.openshift.io/cluster-api-machine-role: <role>` with a user-replaced value `<role>`. Hence we are asking users to specify their value for `<role>`.